es/04_0_Enviando_Transacciones_Bitcoin

```diff
@@ -0,0 +1,30 @@
+# Capítulo Cuatro: Enviando Transacciones en Bitcoin
+
+Este capítulo describe tres metodos diferentes para enviar bitcoins a direcciones tipicas P2PKH desde la línea de comandos, usando unicamente la interfaz de bitcoin-cli.
+
+## Objetivos de este Capítulo
+
+Después de trabajar en este capítulo, un desarrollador podrá:
+
+   * Decidir cómo enviar dinero a través de Bitcoin
+   * Crear una transacción en crudo
+   * Utilizar la aritmética para calcular las tarifas
+
+Con los objetivos complementarios se tendrá la habilidad de: 
+
+   * Entender las transacciones y sus comisiones
+   * Entender las transacciones de tipo Legacy & SegWit
+   * Usar métodos básicos para enviar dinero 
+   * Usar métodos automaticos para calcular comisiones para enviar dinero
+   * Entender el peligro de las transacciones sin procesar
+   
+## Tabla de contenido
+  
+  * [Sección Uno: Enviando Monedas de Forma Sencilla](04_1_Sending_Coins_The_Easy_Way.md)
```

Comment:
`* [Sección Uno: Enviando Monedas de la Forma Fácil](04_0_Enviando_Transacciones_Bitcoin.md)`
to match the titles and links in LEEME.md